assets/js/modules/adsense/components/settings/SettingsForm.js

```diff
@@ -71,6 +78,24 @@ export default function SettingsForm() {
 				checkedMessage={ checkedMessage }
 				uncheckedMessage={ uncheckedMessage }
 			/>
+
+			{ webStoriesActive && (
+				<Fragment>
+					<WebStoriesAdUnitSelect />
+					<p>
+						{ __( 'This ad unit will be used for your Web Stories.', 'google-site-kit' ) }
+						{ ' ' }
+						<Link
+							href={ supportURL }
+							external
+							inherit
+							aria-label={ __( 'Learn more about Ad Sense Web Stories.', 'google-site-kit' ) }
+						>
+							{ __( 'Learn more', 'google-site-kit' ) }
+						</Link>
+					</p>
+				</Fragment>
+			) }
```

Comment:
@benbowler could you please move it above the `UseSnippetSwitch` element to make it look similar to the Analytics settings?